app/models/option_sets/import_data_cleaner.rb

```diff
@@ -95,5 +98,14 @@ def extract_row_metadata(row, row_num, meta_headers)
       end
       metadata
     end
+
+    def detect_translations(headers)
+      languages = []
+      headers.each do |h|
+        language = h.match(/\[(..)\]/)
```

Comment:
I think this may match anywhere within the string, I will make sure it only matches the end of the string. I'll also modify to make it acceptable to use more than 2 characters such as `ar_EG`.

Reply:
thank you Kevin!